### PR TITLE
DSD-666: image bottom margin

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-666](https://jira.nypl.org/browse/DSD-666)

## This PR does the following:

- Adds bottom margin to the `Image` component when used in the `Card` component and the `imageAspectRatio` prop is set to `ImageRatios.Original`.

## How has this been tested?

Locally in Storyboook.

## Accessibility concerns or updates

- None.

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
Please review #819  first as I branched off that branch by accident so the commit/changes look off.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr819-s7jqrcyfuxghw2mpaw2f6fczgmqayfuj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c202bfdc579fc65d475485

File: src/theme/components/card.ts [hunk 1]
```diff
@@ -164,10 +164,14 @@ const CardImage = {
           order: "2",
         }
       : {};
-    return {
-      ...imageAtEndStyles,
-      ...layoutStyles,
-    };
+    return imageAspectRatio === "original"
+      ? {
+          marginBottom: "xs",
```
Comment (bigfishdesign13) [inline src/theme/components/card.ts]: Sorry, I did not correctly specify that this bottom margin only needs to be applied for `ImageRatios.Original` in the `column` layout.  The image in the `row` layout does not need a bottom margin.

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
The update is working, but its scope needs to be narrowed a bit.  Please see my comment.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr819-s7jqrcyfuxghw2mpaw2f6fczgmqayfuj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c202bfdc579fc65d475485

**Review by bigfishdesign13 — APPROVED** [review]
Looks good now.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr819-s7jqrcyfuxghw2mpaw2f6fczgmqayfuj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c202bfdc579fc65d475485

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr819-s7jqrcyfuxghw2mpaw2f6fczgmqayfuj.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61c202bfdc579fc65d475485

**Pull request merged**